understanding/22/target-size-minimum.html

```diff
@@ -50,74 +50,104 @@ <h2>Intent of Target Size (Minimum)</h2>
+              <li><strong>User agent control:</strong> Browsers have default renderings of some controls, such as the days of the month calendar in an <code>input type="date"</code>. As long as the author has not modified the user agent default, the target size for a "User agent control" is excepted.</li>
+              <li><strong>Essential:</strong> If the spacing of the targets is essential to the information being conveyed, the "Essential" exception applies. For example, in digital maps, the position of pins is analogous to the position of places shown on the map. If there are many pins close together, the offset between pins and neighboring pins will often be below 24 CSS pixels. It is essential to show the pins at the correct map location, therefore the Essential exception applies. As well, some jurisdictions legally require online forms to replicate paper forms, which can impose constraints on the size of targets. In such jurisdictions, any legal requirement to replicate small targets can be considered essential.</li>
+          </ol>
+        </section>
 
-        <p>The requirement is independent of the zoom factor of the page; when users zoom in the CSS pixel size of elements does not change. This means that authors cannot meet it by claiming that the target will have enough spacing or sufficient size if the user zooms into the page.</p>
-      
-        <p>The requirement does not apply to targets while they are obscured by content displayed as a result of a user interaction or scripted behavior of content, for example:</p>
+        <section id="size-requirement">
+          <h3>Size requirement</h3>
 
-        <ul>
-          <li>interacting with a combobox shows a dropdown list of suggestions</li>
-          <li>activating a button displays a modal dialog</li>
-          <li>content displays a cookie banner after page load</li>
-          <li>content displays a "Take a survey" dialog after some period of user inactivity</li>
-        </ul>    
-        <p>The requirement does however apply to targets in any new content that appears on top of other content.</p>
-        
-        <p>While the Success Criterion primarily helps touch users by providing target sizing to prevent accidental triggering of adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
+          <p>For a target to be "24 by 24 CSS pixels", it must be possible to draw a 24 by 24 CSS pixel (px) square such that the square is completely within the target.</p>
```

Comment:
I get worried someone may read that "within" as needing to be inside the outer edge of the target, so...
```suggestion
          <p>For a target to be "24 by 24 CSS pixels", it must be possible to draw a 24 by 24 CSS pixel (px) square such that the square is completely within the target (does not extend outside the target's area).</p>
```